src/main/java/com/greenfox/lvlup/controller/PitchDto.java

```diff
@@ -0,0 +1,96 @@
+package com.greenfox.lvlup.controller;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+@JsonSerialize
+public class PitchDto {
+  @NotEmpty(message = "PitchDto name is required.")
+  String badgeName;
+  @NotNull(message = "Old level is required.")
+  Integer oldLVL;
+  @NotNull(message = "Pitched level is required.")
+  Integer pitchedLVL;
+  @NotEmpty(message = "PitchDto message is required.")
+  String pitchMessage;
+  @NotEmpty(message = "Holders are required.")
+  List<String> holders;
+
+  public PitchDto(String badgeName, int oldLVL, int pitchedLVL, String pitchMessage, List<String> holders) {
+    this.badgeName = badgeName;
+    this.oldLVL = oldLVL;
+    this.pitchedLVL = pitchedLVL;
+    this.pitchMessage = pitchMessage;
+    this.holders = holders;
+  }
+
+  public PitchDto(String badgeName, int oldLVL, int pitchedLVL, String pitchMessage) {
+    this.badgeName = badgeName;
+    this.oldLVL = oldLVL;
+    this.pitchedLVL = pitchedLVL;
+    this.pitchMessage = pitchMessage;
+  }
+
+  public PitchDto(String badgeName, int pitchedLVL, String pitchMessage, List<String> holders) {
+    this.badgeName = badgeName;
+    this.pitchedLVL = pitchedLVL;
+    this.pitchMessage = pitchMessage;
+    this.holders = holders;
+  }
+
+
```

Comment:
One line is more than enough to separate different constructors/methods

Reply:
Done.